scripts/discovery/generate_traffic.py

```
@@ -0,0 +1,148 @@
+        ("POST", "/actions/scan"),
+        ("POST", "/actions/scan?force=1"),
+        ("GET", "/actions/generateMediaIndex"),
+
+        # WebSocket (upgrade only; proxy captures the upgrade)
+        # This will 400 without WS upgrade headers but still registers the path
+        ("GET", "/:/websockets/notifications"),
+    ]
+
+    print(f"[TRAFFIC] Hitting {len(endpoints)} endpoints via {base} ...", file=sys.stderr)
+    for method, path in endpoints:
+        status, body = fetch(base, path, method=method, headers=headers)
```

Ruff (0.15.15)

[warning] Unpacked variable `body` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)